src/pages/Schedule.vue

```diff
@@ -1,11 +1,251 @@
+          </div>
+          <div class="block-sm blue">
+            <p>November 20</p>
+            <p>7:00 PM - 8:05 PM</p>
+            <h1>Opening <br /> Ceremony</h1>
+          </div>
+          <div class="block-lg maroon">
+            <p>Nov 21 - Nov 26</p>
+            <p>5:00 PM - 8:00 PM</p>
+            <h1>Online <br /> Workshops</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="block-lg yellow">
+            <p>Nov 27 - Nov 29</p>
+            <p>1:00 PM - 1:00 PM</p>
+            <h1>Hackathon <br /> Proper</h1>
+          </div>
+          <div class="block-sm orange">
+            <p>Nov 29</p>
+            <p>7:00 PM</p>
+            <h1>Top 10 <br /> Finalists</h1>
+          </div>
+          <div class="block-sm green">
+            <p>Nov 30</p>
+            <p>1:00 PM Onwards</p>
+            <h1>Final <br /> Pitching</h1>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="workshop-section">
+      <div class="workshops">
+        <h1>Workshops</h1>
+
+        <DropCard>
+          <div class="drop-header-details" slot="header">
+              <div class="drop-header-details-primary">
+                <h4 class="drop-day">Day 1</h4>
+                <div class="drop-title-speaker">
+                  <h4>Product Pitching</h4>
+                  <p>Richard Cruz</p>
+                </div>
+              </div>
+              <h4 class="drop-time">4:50 - 6:35 PM</h4>
+          </div>
+          <p slot="details">
+            In this session, you will learn the essentials 
+            of creating a cohesive marketing strategy for 
+            your product or service. The objective is to 
+            give you clarity on how best to communicate 
+            your solution through the best channel for 
+            your target market. Topics include persona 
+            creations, insighting, content and distribution 
+            strategies, and the basics of digital marketing.
+          </p>
+        </DropCard>
+
+        <DropCard>
+          <div class="drop-header-details" slot="header">
+              <div class="drop-header-details-primary">
+                <h4 class="drop-day">Day 1</h4>
+                <div class="drop-title-speaker">
+                  <h4>Marketing</h4>
+                  <p>Miggy Azurin</p>
+                </div>
+              </div>
+              <h4 class="drop-time">6:35 - 8:00 PM</h4>
+          </div>
+          <p slot="details">
+            In this session, you will learn the essentials 
+            of creating a cohesive marketing strategy for 
+            your product or service. The objective is to 
+            give you clarity on how best to communicate 
+            your solution through the best channel for 
+            your target market. Topics include persona 
+            creations, insighting, content and distribution 
+            strategies, and the basics of digital marketing.
```

Comment:
I think this would be better to be part of the DropCard component not under the Schedule.vue